cmd/containerd/command/main.go

```diff
@@ -91,6 +94,9 @@ func App() *cli.App {
 		ociHook,
 	}
 	app.Action = func(context *cli.Context) error {
+		if runtime.GOOS == "linux" && os.Geteuid() != 0 {
+			return errors.New("rootless mode requires the daemon to be executed as the mapped root in a user namespace")
```

Comment:
@Callisto13 do you mean removing this check?

Reply:
yes that's the one.
I mean we removed it because we didn't need to enforce it for our rootless case.

Reply:
Sure, I'll rebase and remove the check from this PR when 
https://github.com/opencontainers/runc/pull/1862 gets merged.

Reply:
Cool thanks!

Reply:
hey @AkihiroSuda 
just sharing with you all the hacky changes we made to your pr branch. i've added some notes to explain things but ping me with any questions. using these changes based on your pr, garden was able to use containerd as a non-priv user (for basic create/exec/delete lifecycle, other things may yet fall out).
https://github.com/masters-of-cats/containerd/commit/242e830f2b96f06d0b77f6baee195a74bd242d4d
:)

Reply:
Thanks, will look into 👍 

(off-topic: Is [your ContainerCamp](https://2018.container.camp/uk/speakers/claudia-beresford/) slide deck available online?)

Reply:
the slides should be made available by the organisers when they publish the videos... i think.
or i can just share the slides (they are in google drive) using the email on your profile

Reply:
thanks, could you send me the link to the slides if you don't mind 🙏